dashboard/app/views/levels/levelGroup.js

```diff
@@ -20,86 +20,135 @@ function initLevelGroup(
       // show the Unsubmit button.
       $('.unsubmitButton').show();
     }
+
+    // Set the entire page background to be light grey.
+    $('.full_container').addClass("level_group_readonly");
   }
 
+
+  // Whenever an embedded level notifies us that the user has made a change,
+  // check for any changes in the response set, and if so, attempt to save
+  // these answers.  Saving is throttled to not occur more than once every 20
+  // seconds, and is done as soon as possible ("leading edge"), as well as at
+  // the end of a 20 second period if a change was made before then ("trailing
+  // edge").  Any pending throttled calls are cancelled when we go to a new page
+  // and save for that reason.
+
+  this.throttledSaveAnswers =
+    window.dashboard.utils.throttle(saveAnswers, 20 * 1000, {'leading': true, 'trailing': true});
+
+  var lastResponse = null;
+
+  window.answerChangedFn = function () {
+    var currentResponse = window.getResult().response;
+    if (lastResponse !== null && lastResponse !== currentResponse) {
```

Comment:
Never mind, I see now.  This prevents saving initially when nothing has changed.  Maybe clearer to set `var lastResponse = window.getResult().response;` initially?  Otherwise I think the first answer change won't trigger a save.